Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/ViewBookmarkLoader.h

```diff
@@ -40,11 +41,11 @@ namespace AzToolsFramework
         void SaveBookmarkSettingsFile() override;
         bool SaveLocalBookmark(const ViewBookmark& bookmark, ViewBookmarkType bookmarkType);
         bool SaveLocalBookmarkAtIndex(const ViewBookmark& bookmark, int index);
-        bool SaveSharedBookmark(ViewBookmark& bookmark);
+        bool RemoveLocalBookmarkAtIndex(int index);
+        bool SaveSharedBookmark([[maybe_unused]] ViewBookmark& bookmark);
         bool LoadDefaultLocalViewBookmarks();
 
-        template<typename BookmarkComponentType>
-        BookmarkComponentType* RetrieveBookmarkComponent() const;
+        LocalViewBookmarkComponent* RetrieveLocalViewBookmarkComponent() const;
```

Comment:
nit: This function should not be `const` if it's returning a non-`const` reference (either `LocalViewBookmarkComponent*` should be `const`, or we should drop the `const` qualifier).